### PR TITLE
Fix #496 Improve Makefile and install

### LINKED ISSUE #496: Improve DEV windows install guide
As geokrety contributor, I would like a simple doc on how to install geokrety locally.

Goals are the following
- [x] improve `Makefile` to support [DockerForWindows ](https://docs.docker.com/docker-for-windows/) case
- [x] fix `Makefile`   `composer` target (we've got a single  file right now) 
  - [x] copy `vendor/` files onto website container
- [x] improve `Makefile` to support `phpcs`
- [x] improve `Makefile` to support `test` (phpunit)
- [x] improve Makefile to support `seed` feature (cf. #516 @kumy post)
- [x] check if `Makefile` could support travis checks: crlf,trailing-spaces, utf, phplint
- [x] make `winpty`prefix dynamic depending host OS
- [x] **remove** `gkShell/` ( [DockerToolbox](https://docs.docker.com/toolbox/toolbox_install_windows/) ) : no more supported
- [x] update INSTALL doc

branch: **feature/dockerForWindows**

### PR DESCRIPTION
**Pull request by boly38**
Fix #496 Improve Makefile and install
- add Makefile targets : check, test, seed ...
- remove previous unit tests
- add HealthStateTest
- fix compose web mount
- remove gkShell
- improve doc INSTALL
- improve check scripts exclusions

--
PS/ as you initiate the first docker-compose local, poke @okainov 

I'm sure we coulad always provide a better things at all levels, eg. maybe make travis reusing Makefile targets, etc. But I think we could make such improvement after a merge (to avoid getting long time feature branch)

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -4,34 +4,32 @@ If you note other requirements, feel free to add it here.
...
-     docker-compose -f docker-compose.local.yml up
+Then local instance should be up and running and available at http://localhost:8000/
 
-Now local instance should be up and running and available at http://localhost:8000/
-
-Empty database with proper schema was initialized and exposed on local port `13306`.
+An empty database with proper schema was initialized and exposed on local port `13306`. 
+You could initialize some data using Makefile `seed` target.
 
 Errors are logged in `gk-errory` table in the database
```
Comment (kumy) [inline INSTALL.md]: This is not true anymore in `feature/new-theme`

File: tests/bootstrap.php [hunk 2]
```diff
@@ -1,15 +1,3 @@
...
-
-$dbFile = $testConfig.DIRECTORY_SEPARATOR.'konfig-local.php';
-if (!file_exists($dbFile)) {
-    echo "\n bootstrap - config:$testConfig\n";
-    echo ' XXX Test config expected '.$dbFile."\n";
-    echo "     please run generateTestConfig.sh\n";
-    die;
-}
-require_once 'website/templates/konfig.php';
+include 'vendor/autoload.php';
```
Comment (kumy) [inline tests/bootstrap.php]: Isn't it `../vendor`... ?

File: INSTALL.md (hunk 1 above)
Comment (boly38) [reply INSTALL.md]: ok I ll remove that

File: tests/bootstrap.php (hunk 2 above)
Comment (boly38) [reply tests/bootstrap.php]: good catch, I ll remove bootstrap that is useless

**Pull request merged**